articles/sentinel/notebooks.md

```diff
@@ -42,47 +42,123 @@ We've also released some open-source Jupyter security tools in a package named [
+    | **Contributor** | View, create, edit, or delete (where applicable) assets including notebooks in a workspace. For example, contributors can create a notebook, create or attach a compute cluster, submit a run, and deploy a web service. |
+    | **Owner** | Full access to the workspace, including the ability to view, create, edit, or delete (where applicable) assets in a workspace. Additionally, you can change role assignments. |
+    | **Custom Role** | Allows you to customize access to specific control or data plane operations within a workspace. For example, submitting a run, creating a compute, deploying a model or registering a dataset. |
+
+    >[!IMPORTANT] Role access can be scoped to multiple levels in Azure. For example, someone with owner access to a workspace may not have owner access to the resource group that contains the workspace. For more information, see How Azure RBAC works.
+
+    Currently there are no additional built-in roles that are specific to Azure Machine Learning. For more information on built-in roles, see [Azure built-in roles](https://docs.microsoft.com/azure/role-based-access-control/built-in-roles).
+
+    ### Manage workspace access
 
-1. From the Azure portal, navigate to **Azure Sentinel** > **Threat management** > **Notebooks** and then select **Launch Notebook**.
+    If you're an owner of an Azure ML workspace, you can add and remove roles for the workspace. You can also assign roles to users. Use the following links to discover how to manage access:
+    - [Azure portal UI](../role-based-access-control/role-assignments-portal.md)
+    - [PowerShell](../role-based-access-control/role-assignments-powershell.md)
+    - [Azure CLI](../role-based-access-control/role-assignments-cli.md)
+    - [REST API](../role-based-access-control/role-assignments-rest.md)
+    - [Azure Resource Manager templates](../role-based-access-control/role-assignments-template.md)
 
+    If you have installed the [Azure Machine Learning CLI](https://docs.microsoft.com/azure/machine-learning/reference-azure-machine-learning-cli), you can use CLI commands to assign roles to users:
+
+    ```azurecli-interactive 
+    az ml workspace share -w <workspace_name> -g <resource_group_name> --role <role_name> --user <user_corp_email_address>
+    ```
+
+    The `user` field is the email address of an existing user in the instance of Azure Active Directory where the workspace parent subscription lives. Here is an example of how to use this command:
+
+    ```azurecli-interactive 
+    az ml workspace share -w my_workspace -g my_resource_group --role Contributor --user jdoe@contoson.com
+    ```
+
+    > [!NOTE]
+    > "az ml workspace share" command does not work for federated account by Azure Active Directory B2B. Please use Azure UI portal instead of command.
```

Comment:
```suggestion
    > "az ml workspace share" command does not work for federated account by Azure Active Directory B2B. Please use the Azure UI portal instead of the command line.
```
Also, not sure what you mean by "federated account by Azure Active Directory B2B". Could you please explain that to me?